.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 **/*.rs.bk
 Cargo.lock
+.idea/
```

Comment:
minor but I'd prefer to not introduce this to the git ignore since they are not related to the project.

Reply:
Fixed.